files/grest/rpc/transactions/tx_info.sql

```diff
@@ -471,6 +473,86 @@ BEGIN
             PIC.tx_id = ANY (_tx_id_list)
         ) AS tmp
 
+        GROUP BY tx_id
+        ORDER BY tx_id
+      ),
+
+      _all_native_scripts AS (
+        SELECT
+          tx_id,
+          JSON_AGG(data) AS list
+        FROM (
+          SELECT
+            script.tx_id,
+            JSON_BUILD_OBJECT(
+              'script_hash', ENCODE(script.hash, 'hex'),
+              'script', script.json
+            ) AS data
+          FROM
+            script
+          WHERE
+            script.tx_id = ANY (_tx_id_list)
+            AND
+            script.type IN ('timelock', 'multisig')
+        ) AS tmp
+
+        GROUP BY tx_id
+        ORDER BY tx_id
+      ),
+
+      _all_plutus_contracts AS (
+        SELECT
+          tx_id,
+          JSON_AGG(data) AS list
+        FROM (
+          SELECT
+            redeemer.tx_id,
+            JSON_BUILD_OBJECT(
+              'address', INUTXO.address,
+              'script_hash', ENCODE(script.hash, 'hex'),
```

Comment:
Are these 2 fields the same?

Reply:
One is an address and one is a hash of script. Or did you mean other fields than INUTXO.address and script.hash ?